stages/org.osbuild.oscap.remediation

```diff
@@ -25,6 +27,7 @@ SCHEMA = """
 "properties": {
   "data_dir": {
     "type": "string",
+    "default: "/root",
```

Comment:
You are missing one double-quote at the end of `default`

Reply:
Oh!